docs/nifgen/class.rst

```diff
@@ -5338,11 +5338,11 @@ create_waveform_from_file_f64
                 |
                 | ****Default Value**:** :py:data:`~nifgen.ByteOrder.LITTLE`
 
-                +-------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+                +-------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
needs to be merged from master... (encoding screwing up width problem)

Reply:
Changed